spec/components/nav-list/nav-list.automater.js

```diff
@@ -0,0 +1,122 @@
+const dateParse = require('date-fns/parse');
+const isDateEqual = require('date-fns/is_equal');
+const startOfDay = require('date-fns/start_of_day');
+const webdriver = require('selenium-webdriver');
+
+const seleniumProxy = require('../../selenium-proxy');
+
+const By = webdriver.By;
+
+const statuses = {
+	EXPANDED: 'expanded',
+	COLLAPSED: 'collapsed'
+};
+
+const helpers = (state) => {
+	/**
+	 * Private Methods
+	 */
+	const isPanelOpen = async (elm) => {
+		try {
+			const openPanel = await elm.findElement(By.css(state.options.panelIsOpen));
+			return !!openPanel;
+		}
+		catch (ex) {
+			return false;
+		}
+	};
+
+	const checkNavPanelStatus = async (initialStatus, targetButtonToClickSelector) => {
+		const collapsedButtons = await state.driver.findElements(By.css(targetButtonToClickSelector));
+		if (collapsedButtons && collapsedButtons.length) {
+			const collapsedButton = collapsedButtons[0];
```

Comment:
Why are using index 0? I think I know, but you should either add a comment or make the index a const named something like "firstCollapsedButtonIndex".

Reply:
good call, done.